ros2_generate_interface_docs/ros2_generate_interface_docs/main.py

```diff
@@ -14,11 +14,82 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import argparse
+import os
 import sys
+import time
+
+from ros2_generate_interface_docs import msg_utils
+from ros2_generate_interface_docs import utils
+
+from rosidl_runtime_py import get_message_interfaces
+
+
+def generate_interfaces(interfaces, html_dir, template, interface_type):
+    """
+    Generate the index and documentation for each message.
+
+    Parameters
+    ----------
+    interfaces: dict of {str : str[]}
+        dictionary with the interface package name associated with all the interface
+        for this package.
+    html_dir: str
+        path to the directory to save the generated documentation
+    template: str
+        template name
+    interface_type: str
+        type of the interface: msg, action or srv
+
+    """
+    for package_name, interface_names in sorted(interfaces.items(), key=lambda item: item[0]):
```

Comment:
`, key=lambda item: item[0]` seems unnecessary. Since the keys are unique sorting the tuples as is should have the same result.